workbase/src/renderer/components/Visualiser/VisualiserGraphBuilder.js

```diff
@@ -72,21 +84,28 @@ async function buildLabel(node) {
 
 async function prepareSchemaConcept(schemaConcept) {
   schemaConcept.label = await schemaConcept.label();
+  schemaConcept.attributes = await computeAttributes(schemaConcept);
 }
 
 async function prepareEntity(entity) {
   entity.type = await (await entity.type()).label();
   entity.label = await buildLabel(entity);
+  entity.attributes = await computeAttributes(entity);
+  entity.isInferred = await entity.isInferred();
 }
 
 async function prepareRelationship(rel) {
   rel.type = await (await rel.type()).label();
+  rel.attributes = await computeAttributes(rel);
+  rel.isInferred = await rel.isInferred();
 }
 
 async function prepareAttribute(attribute) {
   attribute.type = await (await attribute.type()).label();
   attribute.value = await attribute.value();
   attribute.label = await buildLabel(attribute);
+  attribute.attributes = await computeAttributes(attribute);
```

Comment:
blocking everything waiting on all the attributes to be loaded might become really heavy, specially with high latency network - it's fine for now but need to implement a lazy loading in background that doesnt block ASAP